### DISCUSSION TITLE
Does node installed as dependency and node installed trough NVM makes trouble?

### DISCUSSION THREAD

**Discussion by mudasar187** [Homebrew/discussions#1710] [Everyday usage] [answered]
I have installed Yarn trough Homebrew, it was a dependency needed for that which was `node`. I have also installed latest --lts version for Node trough NVM. Does it make any trouble? Can i delete the node dependency installed in Homebrew? Does Yarn need node as dependency trough homebrew or can I use the node installed trough NVM?

Thank you for help!

**Comment by SMillerDev** [comment, marked as answer]
> Does Yarn need node as dependency trough homebrew or can I use the node installed trough NVM?

Brew tools always need their dependencies from brew. They have no way to depend on other versions.

**Reply by mudasar187** [reply]
So yarn will not work by ignore node dependency or use the node installled trough NVM?

**Reply by carlocab** [reply]
It depends on how yarn finds Node. If it looks in your `PATH` it might use the Node you have from nvm. No guarantees about that working even if yarn can find the nvm-installed Node, though.
